sonar-application/src/main/java/org/sonar/application/App.java

```diff
@@ -50,6 +50,7 @@
   private final JavaCommandFactory javaCommandFactory;
   private final Monitor monitor;
   private final Supplier<List<JavaCommand>> javaCommandSupplier;
+  private final Cluster cluster;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "cluster" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1068)